packages/packages/commonwell-cert-runner/src/payloads.ts

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+import {
+  AddressUseCodes,
+  IdentifierUseCodes,
+  NameUseCodes,
+  Person,
+} from "@metriport/commonwell-sdk";
+import * as nanoid from "nanoid";
+
+// PERSON
+export const caDriversLicenseUri = "urn:oid:2.16.840.1.113883.4.3.6";
+export const driversLicenseId = nanoid.nanoid();
+const mainDetails = {
+  address: [
+    {
+      use: AddressUseCodes.home,
+      zip: "94041",
+      state: "CA",
+      line: ["335 Pioneer Way"],
+      city: "Mountain View",
+    },
+  ],
+  name: [
+    {
+      use: NameUseCodes.usual,
+      given: ["Paul"],
+      family: ["Greyham"],
+    },
+  ],
+  gender: {
+    code: "M",
+  },
+  birthDate: "1980-04-20T00:00:00Z",
+  identifier: [
+    {
+      use: IdentifierUseCodes.usual,
+      key: driversLicenseId,
+      system: caDriversLicenseUri,
+      period: {
+        start: "1996-04-20T00:00:00Z",
+      },
+    },
+  ],
+}
+
+const secondaryDetails = {
```

Comment:
The reason for main and secondary details is so that the person and patient have matching details when referencing each other